lib/recurly/fraud.js

```diff
@@ -83,8 +84,9 @@ export class Fraud extends Emitter {
       done: (error, response) => {
         debug("response from recurly data collector", error, response);
         if (error) {
-          throw errors('fraud-data-collector-request-failed', { error });
-        } else if (response.content) {
+          const requestFailedError = errors('fraud-data-collector-request-failed', { error });
+          this.emit('error', requestFailedError);
+        } else if (response.content && this.form) {
```

Comment:
Because we're not throwing an error anymore, without checking for `this.form`, an unhandled exception would get thrown from this line https://github.com/recurly/recurly-js/blob/emit-fraud-error/lib/recurly/fraud.js#L103